eos-knowledge-lib.modules

```diff
@@ -199,6 +199,7 @@ use the latest stable release branch from GNOME if you prefer that.
       <dep package="gjs"/>
       <dep package="glib"/>
       <dep package="gtk+"/>
+      <dep package="emeus"/>
```

Comment:
Should the module type just be `meson` in that case? I don't have a meson system dependency here.